includes/pages/user_ical.php

```diff
@@ -61,8 +61,8 @@ function make_ical_entry_from_shift($shift)
     if (isset($shift['Comment'])) {
         $output .= 'DESCRIPTION:' . str_replace("\n", "\\n", $shift['Comment']) . "\r\n";
     }
-    $output .= 'DTSTART;TZID=Europe/Berlin:' . date("Ymd\THis", $shift['start']) . "\r\n";
-    $output .= 'DTEND;TZID=Europe/Berlin:' . date("Ymd\THis", $shift['end']) . "\r\n";
+    $output .= 'DTSTART;TZID=' . config('timezone') . ':' . date("Ymd\THis", $shift['start']) . "\r\n";
+    $output .= 'DTEND;TZID=' . config('timezone') . ':' . date("Ymd\THis", $shift['end']) . "\r\n";
```

Comment:
Please try to convert the time to utc. TZID requires a VTIMEZONE element which simply wont happen.
Also while being at it: Please at DTSTAMP with the same value as the DTSTART. Thanks!